filament/backend/src/vulkan/caching/VulkanPipelineLayoutCache.cpp

```diff
@@ -52,9 +52,8 @@ VkPipelineLayout VulkanPipelineLayoutCache::getLayout(
         }
     }
 
-    auto iter = mPipelineLayouts.find(key);
-    if (iter != mPipelineLayouts.end()) {
-        PipelineLayoutCacheEntry& entry = mPipelineLayouts[key];
+    if (PipelineLayoutMap::iterator iter = mPipelineLayouts.find(key); iter != mPipelineLayouts.end()) {
```

Comment:
nit: this line seems > 100 chars?

Reply:
Oops, I didn't know there was a 100 char rules. I will have to change that file again in the near future, so I can fix that at that time.